resource-timing/resources/iframe-with-delay.sub.html

```diff
@@ -1,3 +1,10 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
 <img src="/xhr/resources/delay.py?ms={{GET[delay]}}" />
+<script>
+    window.addEventListener('load', () => {
+        window.parent.postMessage(
+            {navigationEntry: performance.getEntriesByType('navigation')[0].toJSON(),
```

Comment:
Is the top level frame already checking these numbers? Or is this something you're planning to add in future tests?

Reply:
Oops, I see some files are missing from the commit. Will revise.

Reply:
@yoavweiss re-review? Added the actual tests.